lib/David_tic_tac_toe.rb

```diff
@@ -13,10 +14,14 @@ def self.play
     while true
       game.game_board.print_board
       puts "------------------\n\nPlayer #{current_player.name}'s turn: "
+      puts 'Enter indices separated by  a comma'
       input = gets.chomp
-      indices = input.split(' ', 2)
+      next unless Validation.validate_input(input)
+
+      indices = input.split(',', 2)
       indices = indices.map(&:to_i)
-      next unless game.game_board.valid_position(indices) || game.game_board.board_full
+      # puts indices
```

Comment:
I would remove the debug statements even if they are commented out so that there is less noise in your code.

Reply:
1b8092c74aedfd9c1ae34fa91dee6692b08e5eb5